ggmolvis/utils/node.py

```diff
@@ -2,6 +2,12 @@
 import molecularnodes as mn
 from molecularnodes.blender.nodes import swap, assign_material, add_custom
 
+
+color_dict = {"green": (0, 0.9, 0, 1.0),
+              "red": (1, 0, 0, 1.0),
+              "default": (0, 0, 0, 1.0),
+             }
```

Comment:
I suppose we could support a few defaults, a bit like VMD, but I'm pretty sure it would help for a user to be able to specify the RGBA `tuple` themselves as well, especially with all the resources/websites available for producing nice mixes of such values.